### ISSUE TITLE
internal: Update `just ready` command to match CI

### ISSUE THREAD

**Issue by connorshea** [oxc-project/oxc#15531]
`just ready` is supposed to run all the checks that CI does/ensure your local repo is updated for things like snapshots. I needed to update the cli snaps for a PR, but `just ready` did not catch them or update them.

I only figured out after running into a wall for a bit that `cargo insta test` was the correct way to update the cli snaps to appease CI. We should maybe run `cargo insta test` in `just ready`? Or some equivalent if there's something simpler that'd do the job.

You can reproduce the issue by updating the doc comments for an oxlint CLI flag. Then run `just ready`, and see that the snaps don't update accordingly.

**Comment by Boshen** [comment]
I followed your instructions and `just ready` stops at 

```
stored new snapshot /Users/boshen/oxc/oxc/tasks/website/src/linter/snapshots/cli_terminal.snap.new
stored new snapshot /Users/boshen/oxc/oxc/tasks/website/src/linter/snapshots/cli.snap.new
test linter::cli::test_cli ... FAILED
test linter::cli::test_cli_terminal ... FAILED
```

You need to run `cargo insta accept` but it's not obvious from the outputs, there's only a hint

```
To update snapshots run `cargo insta review`
Stopped on the first failure. Run `cargo insta test` to run all snapshots.
```

The other hint is in https://oxc.rs/docs/contribute/linter.html#snapshot-testing
